js/packages/teams-ai/src/Application.ts

```diff
@@ -68,6 +83,8 @@ export interface Query<TParams extends Record<string, any>> {
 export interface ApplicationOptions<TState extends TurnState> {
     /**
      * Optional. Bot adapter being used.
+     * @deprecated
```

Comment:
We probably want to throw an error or at least set a warning when adapter is being set. The test files set adapter, and it's not a big deal to update but will need to be done, and the reminder/warning will help with that.

Reply:
Would be good to also add migration notes in getting-started since it's a breaking change technically.